ci/do_ci.sh

```diff
@@ -2,7 +2,7 @@
 
 # Run a CI build/test target, e.g. docs, asan.
 
-set -e
+set -ex
```

Comment:
If this is something you'd like to keep, please make a separate PR with this change, since it is unrelated to this PR.